CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 ## vUNRELEASED
 
+#### New features and UX changes
+
+- Ensure that the initial name and labels used for discovery matches the name and labels in the Image Manifest as specified in the appc spec ([#2311](https://github.com/coreos/rkt/pull/2311)). Now users calling `rkt prepare/run/fetch example.com/aci:latest` but receiving an aci with a different version (like v2.0) will receive a validation error. With the current rkt "latest" pattern, users that want to use the "latest" image should use `rkt prepare/run/fetch example.com/aci` or if they want to bypass a locally cached image in the store and fetch an updated image (if the ACI's image discovery provides it) should use `rkt --no-store prepare/run/fetch example.com/aci`.
```

Comment:
s/matches/match/

The last sentence is a bit confusing, also, `--no-store` needs to go after `prepare/run/fetch`. How about:

> Users wanting the latest image should use `rkt prepare/run/fetch example.com/aci` without any labels. If the discovery server supports the "latest" pattern, the user can bypass a locally cached image in the store and fetch an updated image using `rkt prepare/run/fetch --no-store example.com/aci` option.